Exiled.API/Features/Player.cs

```diff
@@ -392,10 +393,15 @@ public RoleType Role
         /// </summary>
         public bool IsReloading => CurrentItem is Firearm firearm && !firearm.Base.AmmoManagerModule.Standby;
 
-        /*/// <summary>
-        /// Gets a value indicating whether or not the player is zooming with a weapon.
+        /// <summary>
+        /// Gets a value indicating whether or not the player is aiming with a weapon.
+        /// </summary>
+        public bool IsAimingWeapon => CurrentItem is Firearm firearm && firearm.Base.AdsModule.ServerAds;
```

Comment:
We should probably make this a property in the Firearm class, then here do 
```cs
public bool IsAimingDownWeapon => CurrentItem is Firearm firearm && firearm.Aiming```

Reply:
ok